integrations/create.go

```diff
@@ -0,0 +1,49 @@
+package integrations
+
+import (
+	"fmt"
+
+	"gopkg.in/errgo.v1"
+
+	"github.com/Scalingo/cli/config"
+)
+
+func Create(scmType string, link string, token string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get Scalingo client")
+	}
+
+	check, err := checkIfIntegrationAlreadyExist(c, scmType)
+	if err != nil {
+		return errgo.Mask(err)
+	}
+	if check {
+		fmt.Printf("Integration '%s' is already linked with your Scalingo account.\n", scmType)
+		return nil
+	}
+
+	if scmType == "github" || scmType == "gitlab" {
```

Comment:
I don't understand, you can't create these integrations with the CLI?... :(

Reply:
No :(, the CLI redirect the user in the URL of the dashboard to make an OAuth dance.

We would have liked to do it. But it would have taken more time and it was decided to not to do it.